### ISSUE TITLE
Getting Error: unable to get local issuer certificate

### ISSUE THREAD

**Issue by VivekTodur** [yagop/node-telegram-bot-api#741]
Hi,
I am getting this error even with the simple example script.

```sh
DEBUG=* node test.js has resulted in the following lines,

node-telegram-bot-api polling with options: {"offset":0,"timeout":10} +0ms
  node-telegram-bot-api HTTP request: {"form":{"offset":0,"timeout":10},"method":"POST","url":"https://api.telegram.org/bot_MY_TOKEN_KEY/getUpdates","simple":false,"resolveWithFullResponse":true,"forever":true} +0ms
  node-telegram-bot-api polling error: EFATAL: Error: unable to get local issuer certificate +978ms
error: [polling_error] {"code":"EFATAL","message":"EFATAL: Error: unable to get local issuer certificate"}
```

**Comment by kamikazechaser** [comment]
Are you using a self signed certificate? If so please check into Telegram API docs on how to correctly configure it.

**Comment by VivekTodur** [comment]
No not using anything... Using as is in the example.. with relevant token
key.

**Comment by stylepatrick** [comment]
Hello, i am getting the same issue. Is there a way to solve it? I am not using a self signed certificate.

**Comment by VivekTodur** [comment]
No, I couldn't able to identify the issue, but it's happening with only node v12. On a machine loaded with node 10 it is working fine. So I assume some issue with node configuration. Please suggest which configuration should be modified to avoid this error on node 12.

Thanks.

**Comment by stylepatrick** [comment]
Hello, thanks for your answer. I will try with an older node version.

**Comment by juandelacruz-calvo** [comment]
For anyone getting to this post in the future. The problem usually relates to the laptop you are using. Some companies install security software that replaces the Server certificates for specific domains in an attempt to obtain the information of the reques.t In my case it was Cisco Umbrella software. So usually executing your code in a different computer that doesn't have that kind of software fixes the issue.

**Comment by yagop** [comment]
This error comes from the runtime TLS trust store, commonly because a corporate HTTPS inspection certificate is not trusted. It is not specific to the Telegram client. Configure the trusted CA, for example with Node's `NODE_EXTRA_CA_CERTS`, and do not disable TLS verification. Please open a new issue with a minimal v2 reproduction if the library itself mishandles a valid certificate chain. Closing as not planned.
